src/containers/Review/assignment/Assignment.tsx

```diff
@@ -1,7 +1,7 @@
 import React, { useState } from 'react'
 import { Segment, Header } from 'semantic-ui-react'
-import { AssignmentDetails, FullStructure } from '../../utils/types'
-import AssignmentSectionRow from '../Assign/AssignmentSectionRow'
+import { AssignmentDetails, FullStructure } from '../../../utils/types'
+import AssignmentSectionRow from '../../Assign/AssignmentSectionRow'
 import ReviewSectionRow from './ReviewSectionRow'
 
 interface ReviewHomeProps {
```

Comment:
Could change this name also, please?

Not sure, because we already have other components called `Assignment` Should this be called AssignmentLevel or something - when we add the other filter?

Reply:
Would it be okay if I move the whole "Assign" folder (in containers) into the "assignment" tab folder?

Reply:
Yes. All good

Reply:
Cool -- 65cca0b2